mono/utils/w32handle.h

```diff
@@ -165,10 +167,11 @@ int
 mono_w32handle_unlock_handle (gpointer handle);
 
 typedef enum {
-	MONO_W32HANDLE_WAIT_RET_SUCCESS_0 =  0,
-	MONO_W32HANDLE_WAIT_RET_ALERTED   = -1,
-	MONO_W32HANDLE_WAIT_RET_TIMEOUT   = -2,
-	MONO_W32HANDLE_WAIT_RET_FAILED    = -3,
+	MONO_W32HANDLE_WAIT_RET_SUCCESS_0   =  0,
+	MONO_W32HANDLE_WAIT_RET_ABANDONED_0 =  0x80,
```

Comment:
I picked 0x80 since it's the value of `WAIT_ABANDONED_0`. But I agree, `MONO_W32HANDLE_WAIT_RET_ABANDONED_0` must be at least `MONO_W32HANDLE_WAIT_RET_SUCCESS_0 + MONO_W32HANDLE_MAXIMUM_WAIT_OBJECTS` so using that explicitly would avoid future bugs if `MONO_W32HANDLE_MAXIMUM_WAIT_OBJECTS` is changed.